tests/fixtures/ssl/client.certificate.pem

```diff
@@ -0,0 +1,89 @@
+Bag Attributes
```

Comment:
Put a README.md in this directory to explain exactly how these files were generated, e.g., openssl command lines and such, so that they can be reproduced if needed (e.g., new openssl version).